components/generic/RightsStatementButton.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <SmartLink
```

Comment:
This results in the icon for external links showing, but in the design, and as clarified during sprint refinement, in this context it is not wanted. Could you come up with a way to not show it here, but to continue showing it in all other contexts, likely via a prop sent down to SmartLink.

Reply:
I have added the hideExternalIcon prop, other smartLinks will still show the external link icon